tips-and-tricks/nvidia_stats.md

```diff
@@ -0,0 +1,29 @@
+# Print Nvidia GPU Stats
+
+It is common to want to see a readout of GPU usage when working on an HPC to test that your scripts are operating as you expect them to.
+One of the simplest ways to do this is to have a second terminal window (a possibility with TMUX), which regularly prints out the GPU statistics. 
+
+A useful command for Nvidia GPUs is `nvidia-smi`, which displays information about the available GPUs and what processes are running.
+The following command prints out some useful GPU statistics formatted into a line per GPU.
+
+```bash
+nvidia-smi -l 1 --query-gpu=index,temperature.gpu,utilization.gpu,memory.used,memory.total --format=csv,noheader,nounits | awk -F ',' '{printf "GPU %d | Temp: %d°C | Util: %3d%% | Mem: %05dMB/%dMB\n", $1, $2, $3, $4, $5}'
```

Comment:
To avoid horizontal scrolling, how would you feel about splitting this command across multiple lines? Something like this maybe?

```suggestion
nvidia-smi -l 1 --query-gpu=index,temperature.gpu,utilization.gpu,memory.used,memory.total \
    --format=csv,noheader,nounits | \
    awk -F ',' '{printf "GPU %d | Temp: %d°C | Util: %3d%% | Mem: %05dMB/%dMB\n", $1, $2, $3, $4, $5}'
```

I know this can be controversial. I'm not keen on splitting lines like this myself, but wide codeblocks in GitHub are quite hard to navigate in my opinion.